pkgs/by-name/tm/tmuxifier/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchFromGitHub, installShellFiles }:
+
+stdenv.mkDerivation rec {
+  pname = "tmuxifier";
+  version = "0.13.0";
+
+  src = fetchFromGitHub {
+    owner = "jimeh";
+    repo = "tmuxifier";
+    rev = "v${version}";
+    sha256 = "sha256-7TvJnvtZEo5h45PcSy3tJN09UblswV0mQbTaKjgLyqw=";
+  };
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    sed -i "s@set -e@TMUXIFIER=$out/share\nTMUXIFIER_LAYOUT_PATH=\"\$HOME/.tmuxifier-layouts\"\nset -e@" bin/tmuxifier
+
+    # Create directories.
+    install -m 755 -d $out/bin/
+    install -m 755 -d $out/share/init
+
+    # Perform installation.
+    install -m 755 bin/tmuxifier $out/bin/
+    install -m 644 init.fish init.sh init.tcsh $out/share/init
+    cp -r lib libexec templates $out/share
```

Comment:
We should probably use $out/lib and $out/libexec

Reply:
I tried this, but it breaks the tmuxifier if you install all or at least one $out/lib and $out/libexec

```
~/src/github.com/NixOS/nixpkgs$ result/bin/tmuxifier s mysession
result/bin/tmuxifier: line 34: /nix/store/qqbv9pjp2839rlzgrna1k7r8iv1ylb11-tmuxifier-0.13.0/share/lib/env.sh: No such file or directory

~/src/github.com/NixOS/nixpkgs$ result/bin/tmuxifier s mysession
result/bin/tmuxifier: line 41: tmuxifier-tmux-version: command not found
result/bin/tmuxifier: line 59: tmuxifier-resolve-command-path: command not found
tmuxifier: no such command 's'
```

Reply:
Then we should patch tmuxifier to use the correct path.